section2/exercises/if-statements.js

```diff
@@ -43,7 +43,13 @@ Log to the console the appropriate age range
 
 var dogAge = 3;
 // Write your conditional here
-
+if (dogAge <= 2) {
+  console.log('puppy');
+} else if (dogAge >= 3 || dogAge <= 9) {
+  console.log('adult');
+} else if (dogAge >= 10) {
```

Comment:
Same thing here. It can really just be else, because the only values left will be greater than or equal to 10.